readme.md

```diff
@@ -18,52 +18,174 @@ For API reference, please see the
 [H3 Documentation](https://h3geo.org/).
 
 **NOTE: This version (v.3.6.1) corresponds to the new Cython
-bindings, which will replace the old bindings (v3.4.3).
-This version is not yet published to PyPI.**
+bindings, which have replaced the old bindings (v3.4.3).**
 
 
 ## Install from PyPI
 
-TODO: instructions for installing from pre-built wheels
+`pip install h3`
 
+**New in v3.6.1**: We upload pre-built
+[Python Wheels to PyPI](https://pypi.org/project/h3) for Linux/Mac/Windows,
+which should avoid many previous installation issues.
 
-## Install from GitHub
+Try the library with:
```

Comment:
Nit: I'd usually go with `## Usage` here.

Reply:
How about just `Usage:`. I don't know if it needs a whole section to itself.

Reply:
Up to you - it's the first thing I look for in most READMEs, so it doesn't hurt to make it big :)

Reply:
I approved because I assumed @nrabinowitz 's suggestion here was going to be taken. Hard agree on `## Usage`. It's the convention for nearly all GitHub projects.

Reply:
updated!